ProcessHacker/prpggen.c

```diff
@@ -93,6 +93,55 @@ PPH_STRING PhGetProcessItemProtectionText(
     return PhCreateString(L"N/A");
 }
 
+PPH_STRING PhGetProcessItemImageTypeText (
+    _In_ PPH_PROCESS_ITEM ProcessItem
+    )
+{
+    PWSTR arch;
+    PWSTR bits;
+
+    switch (ProcessItem->Architecture)
+    {
+    case IMAGE_FILE_MACHINE_I386:
+        {
+            arch = L"I386 ";
+            break;
+        }
+    case IMAGE_FILE_MACHINE_AMD64:
+        {
+            arch = L"AMD64 ";
+            break;
+        }
+    case IMAGE_FILE_MACHINE_ARM:
+        {
+            arch = L"ARM ";
+            break;
+        }
+    case IMAGE_FILE_MACHINE_ARM64:
+        {
+            arch = L"ARM64 ";
+            break;
+        }
+    case IMAGE_FILE_MACHINE_UNKNOWN:
+    default:
+        {
+            arch = L"";
+            break;
+        }
+    }
+
+    if (ProcessItem->IsWow64Valid)
```

Comment:
This needs corrected for 32bit OS. And should set `N/A` when both arch and bits are empty for consistency.